analyzer/restapi-plugin/src/main/java/eu/fasten/analyzer/restapiplugin/mvn/GraphResolverInitializer.java

```diff
@@ -0,0 +1,15 @@
+package eu.fasten.analyzer.restapiplugin;
+
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+
+public class GraphResolverInitializer {
+
+    public static void sendRequestToInitGraphResolver() {
+        RestTemplate restTemplate = new RestTemplate();
+        String url = "http://localhost:8080/packages/package/0/resolve/dependents?transitive=false";
```

Comment:
It's not a good idea to have this hardcoded URL here.

Reply:
Previously, we sent this get request with the hard-coded url to init the grah maven resolver: ```http://localhost:8080/__INTERNAL__/packages/0/directedgraph``` , so to be honest I followed the same logic. The reasoning of this initialization is explained briefly on this [comment](https://github.com/fasten-project/fasten/pull/351/files#r762173190). If we want to hide this information, I would also suggest removing the hard-coded server port from this [line](https://github.com/fasten-project/fasten/blob/0af2d600b5da89fe4ecc77b707d86c9f0e1a6bb2/analyzer/restapi-plugin/src/main/resources/application.properties#L4).

Reply:
I understand. Perhaps we could at least get the url out into the application.properties file?